src/megatron/energon/flavors/webdataset/fastseek/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
```

Comment:
I'd suggest to move this to a more top-level package. Maybe under `megatron.energon.video`?

Reply:
ack - will do

Reply:
done - i went with `megatron.energon.av` as we're not just doing video. Is that clear enough?